### PR TITLE
replace connect modal for connect button on new position page

### PR DESCRIPTION
**Pull request by zielvna**

### CODE REVIEW

**Review by p6te — COMMENTED** [review]
image
image
connect wallet button is too high

File: src/components/NewPosition/DepositSelector/style.ts [hunk 1]
```diff
@@ -81,6 +81,15 @@ export const useStyles = makeStyles()(theme => {
           filter: 'none'
         }
       }
+    },
+    connectWalletButton: {
+      height: '48px !important',
```
Comment (p6te) [inline src/components/NewPosition/DepositSelector/style.ts:86]: is it possible to not use `!important`?

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Jumping position of button: "Connect wallet" and "Add position"
[Screencast from 19.09.2024 12:44:52.webm](https://github.com/user-attachments/assets/220c1fbf-7f13-4451-b95d-5afd02b82c52)

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
The placeholder inside the input token amount should appear like this when the wallet is disconnected: `Balance: - USDC`

It should be similar to this, but **with the selected token ticker**:
image

Instead of the current version:
image

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Is there a need to trigger a loader for the token balance in case the wallet is disconnected? There should always be a -" value:
[Screencast from 22.09.2024 22:44:52.webm](https://github.com/user-attachments/assets/97ef9bd5-5e73-405f-97fc-311a7be45123)

**Review by p6te — APPROVED** [review]

**Review by p6te — APPROVED** [review]

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Bug with token balances after triggering refresh:
[Screencast from 23.09.2024 14:50:20.webm](https://github.com/user-attachments/assets/21081cd9-863a-491b-ba4b-3778fe497bbf)

**Review by p6te — APPROVED** [review]

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**